src/universal/components/TaskList/TaskList.js

```diff
@@ -31,6 +31,8 @@ const exampleItems = [
 ];
 
 @cssModules(styles)
+// for the decorators
+// eslint-disable-next-line react/prefer-stateless-function
 export default class TaskList extends Component {
   render() {
     const handleCheckboxChanged = (id) => {
```

Comment:
declaring funcs is render is bad news bears. this should be a method. then, we won't have to call an anonymous func, but can call it directly since it'll be in the class context.